Subject: [PATCH] patch fix
---
Index: kmem/src/androidMain/kotlin/korlibs/memory/internal/currentAndroid.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/kmem/src/androidMain/kotlin/korlibs/memory/internal/currentAndroid.kt b/kmem/src/androidMain/kotlin/korlibs/memory/internal/currentAndroid.kt
--- a/kmem/src/androidMain/kotlin/korlibs/memory/internal/currentAndroid.kt	(revision 7c90271b23efaf647e73216a9cd4035cdb6de454)
+++ b/kmem/src/androidMain/kotlin/korlibs/memory/internal/currentAndroid.kt	(revision 105efc190948d0334c260b69b14c36ee7444128b)
@@ -1,6 +1,7 @@
 package korlibs.memory.internal
 
 import android.os.Build
+import com.soywiz.kmem.*
 import korlibs.memory.Arch
 import korlibs.memory.Os
 import korlibs.memory.Runtime
@@ -31,8 +32,8 @@
     return@lazy Arch.UNKNOWN
 }
 
-internal actual val currentIsDebug: Boolean get() = korlibs.memory.BuildConfig.DEBUG
+internal actual val currentIsDebug: Boolean get() = BuildConfig.DEBUG
 internal actual val currentIsLittleEndian: Boolean get() = ByteOrder.nativeOrder() == ByteOrder.LITTLE_ENDIAN
 
 internal actual val currentRawPlatformName: String = "android-$currentOs-$currentArch-$currentBuildVariant"
-internal actual val currentRawOsName: String = System.getProperty("os.name") ?: "android"
\ No newline at end of file
+internal actual val currentRawOsName: String = System.getProperty("os.name") ?: "android"
